pynblint/repository.py

```diff
@@ -67,6 +70,50 @@ def large_file_paths(self) -> List[Path]:
                     large_files.append(file_path)
         return large_files
 
+    def _get_dependencies(self) -> set:
+        dependencies = set()
+        for root, dirs, files in os.walk(self.path):
+            for f in files:
+                # check if exist a requiremets.txt file
+                if f.endswith("requirements.txt"):
+                    try:
+                        with open((Path(root) / Path(f)), "r") as fi:
+                            file_row = fi.read()
+                            # print(file_row)
+                            tmp = file_row.split("\n")
+                            for item in tmp:
+                                dependencies.add(item)
+                            # dependencies.add(file_row)
+                            # dependencies.add(file_row)
```

Comment:
When you parse a package from `requirements.txt`, make sure to strip the version constraint information (e.g., the minimum version number)